manifests/3/3CX/3CXCallFlowDesigner/16.6.53/3CX.3CXCallFlowDesigner.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: 3CX.3CXCallFlowDesigner
+PackageVersion: 16.6.53
+PackageLocale: en-US
+Publisher: 3CX
```

```suggestion
Publisher: 3CX Ltd
```
Needs to match what control panel sees